packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/fabric-v2-2-x/run-transaction-endpoint-v1.test.ts

```diff
@@ -174,5 +176,22 @@ test("runs tx on a Fabric v2.2.0 ledger", async (t: Test) => {
     t.equal(car277.Record.owner, carOwner, `Car has expected owner OK`);
   }
 
+  // await new Promise((resolve) => setTimeout(resolve, 200000));
+  {
+    const promExporter = plugin.getPrometheusExporter();
+    const res = await apiClient.getPrometheusExporterMetricsEndpointV1({
+      promExporter: promExporter,
```

Comment:
@jagpreetsinghsasan Same as above, if the endpoint is now `HTTP GET` then the request body is not actually being used IIRC and therefore can be removed altogether.

Reply:
Fixed